### DISCUSSION TITLE
Welcome to esp_canbus Discussions!

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp_canbus#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HelmiSchadi** [comment]
Is there any installation guide for beginners. :-)

**Reply by nliaudat** [reply]
This is not intended for beginners :)

**Comment by bagges** [comment]
I just tried the latest version of the code and it seems to work really good. Thank you guys for this awesome work!

**Comment by jot-koehler** [comment]
Interesting approach. Unfortunately, I only have the basic BG02 E control unit. Can I also place this device between it and my ventilation system (HomeVent 350) in order to communicate with the system?

**Reply by abichinger** [reply]
<img width="1080" height="1434" alt="image" src="https://github.com/user-attachments/assets/a1026ec2-add8-4726-91e4-8196a0e72383" />

I found this image on [reddit](https://www.reddit.com/r/homeassistant/comments/1jo0i68/make_this_homevent_smart/)

It looks as if the module is using the same RJ45 connector as the other control module. Therefore, I assume it should work.

**Reply by jot-koehler** [reply]
Thanks! Seems that the simple control unit is analogue and powered with 24 Volts. But i can propably connect to the CAN-bus directly on the HomeVent’s platine and use both parallel. I will check this in the next few days.

**Reply by nliaudat** [reply]
warning, the power input is for 12V canbus only

**Reply by jot-koehler** [reply]
Yes. But thanks. :)

**Reply by jot-koehler** [reply]
I guess there a two seperate connectors for CAN-bus and analogue on the board. We‘ll see.

**Reply by jot-koehler** [reply]
My Hoval is missing the CAN-bus connectors. 😒

**Comment by nliaudat** [comment]
This gateway works exactly the same as https://github.com/wladwnt/CAN-Gateway

So supported devices should be : https://github.com/wladwnt/CAN-Gateway/wiki/Supported-Devices
and unsupported : https://github.com/wladwnt/CAN-Gateway/wiki/Unsupported-Devices
